.github/workflows/ci.yml

```diff
@@ -53,3 +53,18 @@ jobs:
         with:
           name: package
           path: ./pkg/**/*
+
+  all:
+    if: always()
+    runs-on: ubuntu-latest
+    needs:
+      - sanity-checks
+      - default
+      - main
```

Comment:
These names might be hard to understand what they do, what do you think by renaming them?

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1378/commits/57cc8ac0fccef59aafe85068671377caa91409ab

Reply:
https://github.com/elastic/apm-agent-ruby/pull/1378/commits/294de7afe876da59c9666fb51d28b17f8aedb21b